src/test/integration/assessment/originalDataPoint.ts

```diff
@@ -47,7 +50,11 @@ export default (): void =>
 
     it('Edit existing/not existing Original data point', async () => {
       const editedOriginalDataPoint = await CycleDataController.updateOriginalDataPoint({
-        assessment,
+        assessment: {
+          ...assessment,
+          // Test assessment meta cache is empty
+          metaCache: { ...assessment.metaCache, calculations: { dependants: {}, dependencies: {} } },
+        },
```

Comment:
cant this be added to the test assessment directly ?

Reply:
assessment fetched from db, should we put it in db..?

https://github.com/openforis/fra-platform/blob/e2031fc11b7e832653914f6545f59a1081fce7ff/src/test/integration/assessment/originalDataPoint.ts#L21